src/pages/SignUp/index.js

```diff
@@ -1,31 +1,23 @@
 import React from 'react'
 
 import Header from '../../components/Header'
+import Layout from '../../components/Layout'
+import { H1 } from '../../components/Typography'
+import InputField from '../../components/Input'
+import { Button } from '../../components/Button/styled'
 
 const SignUp = () => (
-  <>
+  <Layout>
     <Header />
```

Comment:
Header occurs in each component, where is layout (so they are bound together). Move header component into Layout component

Reply:
Ok, Done! Thanks